DuckDuckGo/BarsAnimator.swift

```diff
@@ -84,10 +84,6 @@ class BarsAnimator {
         let ratio = calculateTransitionRatio(for: scrollView.contentOffset.y)
         delegate?.setBarsVisibility(1.0 - ratio, animated: false)
         transitionProgress = ratio
-
-        var offset = scrollView.contentOffset
-        offset.y = transitionStartPosY
-        scrollView.contentOffset = offset
```

Comment:
adjusting the content offset while it was scrolling is what causes the page to stop scrolling while the bars are being hidden, which is not what other browsers do and at least one person internally has complained about

Reply:
Do you know why we were doing this?

Reply:
Not really, but it's been flagged as a bug since and does seem inconsistent with other browsers and personally I think it is a bit annoying :)